weaviate/connect/connection.py

```diff
@@ -6,14 +6,20 @@
 import datetime
 import os
 import time
+
+# request library uses simplejson (and its JSONDecodeError) if it is installed on the system and stdlib json otherwise.
+# See https://github.com/psf/requests/issues/4842 for more infos
+try:
+    from simplejson import JSONDecodeError
+except ImportError:
+    from json import JSONDecodeError
```

Comment:
Why not importing `requests.JSONDecodeError` without try except?
https://requests.readthedocs.io/en/latest/api/#requests.JSONDecodeError

Reply:
because nobody told me!